### DISCUSSION TITLE
Cameras with Field Book:  exposure control, image formats, and distortion correction

### DISCUSSION THREAD

**Discussion by edwinreidel** [PhenoApps/Field-Book#783] [Q&A] [answered]
I'm excited about the possibility of using an external camera for phenotyping.  Like most tablets, my **Galaxy Tab A8** has an inferior, 8 MP camera.  Also, I noticed that access to advanced, "Pro Mode" options was lost when acquiring images via Field Book with the tablet:
- no ability to select white balance/color temperature
- no ability to select ISO/sensitivity
- Voice-activated picture taking was unavailable.  This is a small inconvenience, although it's easier to keep the camera level relative to the subject if steadying with two hands

We are thinking about moving to Android phones such as the **Google Pixel** range with great cameras.  The **GoPro** is also an interesting option.  However, the pronounced **_barrel distortion_** that is marketed as a feature of the GoPro would seem to make it unviable for phenotyping applications involving image analysis.  I don't have a GoPro at the moment, so I'm wondering if anyone can answer: 

- Are the **_ProTune_** options for adjusting white balance, sensitivity, and exposure available when shooting with Field Book?
- Is **_Linear FOV_** mode available in Photo mode to correct the barrel distortion?
- Is **_RAW_** output available?
- Is voice activated shutter available?

I'd love to see some sample images!

**Comment by trife** [comment, marked as answer]
>  Are the ProTune options for adjusting white balance, sensitivity, and exposure available when shooting with Field Book?

If these are set on the GoPro, I'd assume that they would carry forward through the images that were captured.

> Is Linear FOV mode available in Photo mode to correct the barrel distortion?

Yes.

> Is RAW output available?

Not sure. @chaneylc ?

> Is voice activated shutter available?

Not currently
